osmomath/math.go

```diff
@@ -15,6 +15,8 @@ var powPrecision, _ = sdk.NewDecFromStr("0.00000001")
 var zero sdk.Dec = sdk.ZeroDec()
 
 var (
+	Tick = sdk.MustNewDecFromStr("1.0001")
```

Comment:
Why are we doing logs respective to this base, rather than just 2

Reply:
I'm tracking this as one of the "Blocking TODOs" in the PR description to revisit next.

I've been following uniswap whitepaper. However, I've also come to the conclusion that we don't need this choice of base and plan to choose the new one while working on overflow tests.

With the base of 2, I've also run into overflow problems so I would like to tackle the correct choice of base next in a separate PR.

Reply:
Changed base to 2 but still might need to revisit once overflow tests are added

Reply:
SG! (FWIW im not at all married to the base of 2! Just if we don't have a reason for any particular base, or some number being superior to 2, would prefer to default to 2)